content/v2.3/upgrade-notes/v2.3.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Upgrade to 2.3
+order: 10
+---
+
+These notes cover an upgrade from 2.2 to 2.3.
+
+## Update the app
+
+- Edit `Gemfile` and update the versions of the Hanami gems to `"~> 2.3"`.
```

Comment:
are we sure we don't want to suggest `"~> 2.3.0"` so that they're not automatically updated to `[>= 2.3, < 3.0]`, or is that what we want?

Reply:
Good point!

Reply:
```suggestion
- Edit `Gemfile` and update the versions of the Hanami gems to `"~> 2.3.0"`.
```